v2/datastream-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/DataStreamToSpannerSimpleIT.java

```diff
@@ -248,4 +464,288 @@ private void assertMovieTableContents() {
+        .hasRecordsUnorderedCaseInsensitiveColumns(events);
+  }
+
+  private void assertAllDatatypeColumns2TableCdcContents() {
+    List<Map<String, Object>> events = new ArrayList<>();
+
+    Map<String, Object> row = new HashMap<>();
+    row.put("varchar_column", "value1");
+    row.put("tinyint_column", 15);
+    row.put("text_column", "text1");
+    row.put("date_column", "2024-02-08");
+    row.put("smallint_column", 50);
+    row.put("mediumint_column", 1000);
+    row.put("int_column", 50000);
+    row.put("bigint_column", 987654321);
+    row.put("float_column", 45.67);
+    row.put("double_column", 123.789);
+    row.put("datetime_column", "2024-02-08T08:15:30Z");
+    row.put("timestamp_column", "2024-02-08T08:15:30Z");
+    row.put("time_column", "29730000000");
+    row.put("year_column", "2022");
+    row.put("char_column", "char_1");
+    // Source column value: 74696e79626c6f625f646174615f31 ( in BYTES, "tinyblob_data_1" in STRING)
+    // results in dGlueWJsb2JfZGF0YV8x base64 encoded string
+    row.put("tinyblob_column", "dGlueWJsb2JfZGF0YV8x");
+    row.put("tinytext_column", "tinytext_data_1");
+    row.put("blob_column", "YmxvYl9kYXRhXzE=");
+    row.put("mediumblob_column", "bWVkaXVtYmxvYl9kYXRhXzE=");
+    row.put("mediumtext_column", "mediumtext_data_1");
+    row.put("longblob_column", "bG9uZ2Jsb2JfZGF0YV8x");
+    row.put("longtext_column", "longtext_data_1");
+    row.put("enum_column", "2");
+    row.put("bool_column", false);
+    row.put("binary_column", "YmluYXJ5X2RhdGFfMQAAAAAAAAA=");
+    row.put("varbinary_column", "dmFyYmluYXJ5X2RhdGFfMQ==");
+    row.put("bit_column", "AQI=");
+    events.add(row);
+
+    SpannerAsserts.assertThatStructs(
+            spannerResourceManager.runQuery(
+                "select varchar_column, tinyint_column, text_column, date_column"
+                    + ", smallint_column, mediumint_column, int_column, bigint_column, float_column"
+                    + ", double_column, datetime_column, timestamp_column, time_column, year_column, char_column"
+                    + ", tinyblob_column, tinytext_column, blob_column, mediumblob_column, mediumtext_column, "
+                    + " longblob_column, longtext_column, enum_column, bool_column, binary_column"
+                    + ", varbinary_column, bit_column from AllDatatypeColumns2"))
+        .hasRecordsUnorderedCaseInsensitiveColumns(events);
+  }
+
+  private void assertCategoryTableBackfillContents() {
+    List<Map<String, Object>> events = new ArrayList<>();
+
+    Map<String, Object> row1 = new HashMap<>();
+    row1.put("category_id", 1);
+    row1.put("full_name", "xyz");
+
+    Map<String, Object> row2 = new HashMap<>();
+    row2.put("category_id", 2);
+    row2.put("full_name", "abc");
+
+    events.add(row1);
+    events.add(row2);
+
+    SpannerAsserts.assertThatStructs(
+            spannerResourceManager.runQuery("select category_id, full_name from Category"))
```

Comment:
Since this is a column rename and column drop test, lets do `select * from Category` here.